### ISSUE TITLE
ServiceWorker Works not correct

### ISSUE THREAD

**Issue by mkozachok** [gdg-x/hoverboard#482]
```
Uncaught TypeError: Cannot read property 'resetLayout' of undefined
    at HTMLElement.<anonymous> (hoverboard-app.html:84)
    at hoverboard-app.html:23
Uncaught Error: The installing service worker became redundant.
    at ServiceWorker.e.onstatechange (service-worker-registration.js:1)
```

After updating app, we still see old one. Only in private tab we are able to see it.

**Comment by ozasadnyy** [comment]
@mkozachok please provide steps to reproduce.
We don't have any issues on DevFest Ukraine website while updating pretty often.
After each build, if you change any file generated hash should be different in SW file. Could you check if it's happening?

**Comment by mkozachok** [comment]
@ozasadnyy yes, hash changes. How to disable this feature? I removed SW file, but this did not help.

**Comment by ozasadnyy** [comment]
@mkozachok try to remove this line https://github.com/gdg-x/hoverboard/blob/master/index.html#L135

The default behavior is to show toast when SW removed old files and cached new:  https://github.com/gdg-x/hoverboard/blob/master/scripts/service-worker-registration.js#L25

**Comment by mkozachok** [comment]
Also tried this, no result. The same behavior.

**Comment by mkozachok** [comment]
@ozasadnyy  this helped to clear caches https://stackoverflow.com/questions/45467842/how-to-clear-cache-of-service-worker

**Comment by ozasadnyy** [comment]
Would you like to send PR?

**Comment by ozasadnyy** [comment]
The issue with updating SW should be fixed in #491
